ext/sockets/tests/socket_sendto_zerocopy.phpt

```diff
@@ -4,9 +4,13 @@ Test socket_sendto with MSG_ZEROCOPY
 sockets
 --SKIPIF--
 <?php
+$arch = php_uname('m');
 if (!defined("SO_ZEROCOPY")) {
     die('skip SO_ZEROCOPY');
 }
+if (strpos($arch, 'ppc') !== false || strpos($arch, 'powerpc') !== false) {
+    die('skip on powerpc arch');
```

Comment:
That would print "reason: on powerpc arch". So it's probably better to reword to something like "skip not for powerpc arch".